R/rdeephaven/man/TableHandle.Rd

```diff
@@ -7,8 +7,54 @@
 \description{
 A TableHandle holds a reference to a Deephaven Table on the server, and provides methods for operating on that table.
 Note that TableHandles should not be instantiated directly by user code, but rather by server calls accessible from
-the `Client` class. See `?Client` for more information.
+the \code{\link{Client}} class. See \code{?Client} for more information.
 }
+\section{Naming tables on the server}{
+
+When a TableHandle is created, it is not automatically bound to a variable name on the server. This means that the
```

Comment:
I have read this same text earlier; consider replacing the full explanation repeated here with a summary (short explanation)  and a pointer to the other part of the documentation with the full explanation.